modules/nodes-pods-vertical-autoscaler-using-about.adoc

```diff
@@ -91,6 +91,39 @@ The output shows the recommended resources, `target`, the minimum recommended re
 
 The VPA uses the `lowerBound` and `upperBound` values to determine if a pod needs to be updated. If a pod has resource requests below the `lowerBound` values or above the `upperBound` values, the VPA terminates and recreates the pod with the `target` values.
 
+[id="nodes-pods-vertical-autoscaler-using-one-pod_{context}"]
+== About changing the VPA minimum value
+
+By default, workload objects must specify a minimum of two replicas in order for the VPA to monitor and update pods. As a result, workload objects that specify fewer than two replicas are not acted upon by the VPA. You can change this minimum value by modifying the `VerticalPodAutoscalerController` object to add the `minReplicas` parameter.
+
+For example, if `minReplicas: 3`, the VPA does not delete and update pods for workload objects that specify fewer than three relicas.  
+
+[NOTE]
+====
+If you set if `minReplicas: 1`, the VPA can delete the only pod for a workload object that specifies only one replica. You should use this setting with one-replica objects only if your workload can tolerate downtime whenever the VPA deletes a pod to adjust its resources. To avoid unwanted downtime with one-replica objects, use the VPA with the `podUpdatePolicy` set to `Initial`, which automatically updates pods only when they are restarted by some process external to the VPA, or `Off`, which allows you to update the pods manually at an appropriate time for your applications. 
```

Comment:
This note is really good. The only improvement I'd like to see is if you could somehow make it clear that since this setting is cluster-wide, individual VPA objects have to opt out of updating. I can't think of  a good way to say it, but I think fewer admins will get it wrong if you can somehow make the contrast clear of cluster setting of minReplicas vs. individual VPA setting of podUpdatePolicy.

Reply:
@joelsmith I added _cluster-wide_ above:  _You can change this cluster-wide minimum value..._
and changed the note to state: _To avoid unwanted downtime with one-replica objects, configure the VPA CRs with the `podUpdatePolicy` set..._
Do you think this help clarify?